### PR TITLE
en: Improve deploy tidb monitor docs

### PR DESCRIPTION
**Pull request by Joyinqin**
### Which TiDB Operator version(s) do your changes apply to? (Required)
improve docs about deploy tidb monitor

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/810
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Review by ran-huang — COMMENTED** [review]
Rest LGTM. Thank you~

File: en/monitor-a-tidb-cluster.md [hunk 1]
```diff
@@ -92,7 +90,7 @@ When Prometheus is deployed with a TiDB cluster, some default alert rules are au
```
[the remark this answers is not in the document]
Comment (Joyinqin) [reply en/monitor-a-tidb-cluster.md:90]: PTAL again, thx~ @ran-huang

**Comment by ran-huang** [comment]
@Joyinqin Can you please address this https://github.com/pingcap/docs-tidb-operator/pull/810#discussion_r520398884 in your PR?

**Comment by ran-huang** [comment]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**